kernel/static/loader/empty-scenes-halloween/contents/EP_01.js

```diff
@@ -0,0 +1,71 @@
+
```

Comment:
These bunch of scenes are exactly the same as the original empty parcels. The only real changes are in the main GLB files.